components/layout/fragment.rs

```diff
@@ -996,6 +1002,7 @@ impl Fragment {
             specific: info,
             inline_context: self.inline_context.clone(),
             pseudo: self.pseudo.clone(),
+            flags: FragmentFlags::empty(),
```

Comment:
We have a unit test that verifies this: https://dxr.mozilla.org/servo/source/tests/unit/layout/size_of.rs#28